### PR TITLE
chore: fix NOTICE file & add license header

### PR DESCRIPTION
**Pull request by imbajin**
due to this PR is too big to review, separate it to 5 commits, coud use `rebase & merge` for it

### CODE REVIEW

**Review by zyxxoo — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Apache License
+	                                 Apache License
```
Comment (javeme) [inline LICENSE:1]: is there a tab:
截屏2023-01-13 23 32 39

**Review by javeme — COMMENTED** [review]
```java
INFO Totally checked 1554 files, valid: 595, invalid: 1, ignored: 958, fixed: 0 
WARNING Failed to determine the comment style of file: NOTICE 
WARNING Failed to determine the comment style of file: NOTICE 
ERROR the following files don't have a valid license header: 
NOTICE 
ERROR one or more files does not have a valid license header 
```

**Comment by imbajin** [comment]
> ```java
> INFO Totally checked 1554 files, valid: 595, invalid: 1, ignored: 958, fixed: 0 
> WARNING Failed to determine the comment style of file: NOTICE 
> WARNING Failed to determine the comment style of file: NOTICE,
> ERROR the following files don't have a valid license header: 
> NOTICE 
> ERROR one or more files does not have a valid license header 
> ```

this already fixed in `release-1.0.0`, could ignore it or merged  #400 first

File: LICENSE (hunk 1 above)
Comment (imbajin) [reply LICENSE]: yes, also found in exist file `dist/LICENSE`, copy from it:) 

should also remove it in other repo

**Review by javeme — APPROVED** [review]

**Review by zyxxoo — APPROVED** [review]

**Pull request merged**